entities/Rules/index.js

```diff
@@ -380,22 +374,22 @@ module.exports = {
       const propSelection = require('../PlayerActions').prompt(
         { notify, UI },
         gameState,
-        [...mortgageAbleProps.map((p) => p.name), 'CANCEL']
+        [...mortgageAbleProps.map((p) => p.name.toUpperCase()), 'CANCEL']
```

Comment:
Point-free would be:
```javascript
...mortgageAbleProps.map(String.prototype.toUpperCase.call)
```
Tbh, requiring the `prototype` and `call` in there can make it feel a lil' more unreadable, mais ouais, not sure what your preference would be

Reply:
Going to leave this for posteriority. I think this is okay to leave as is.